src/Compilers/VisualBasic/Portable/Generated/Syntax.xml.Internal.Generated.vb

```diff
@@ -1,5 +1,5 @@
 ﻿' Definition of syntax model.
-' Generated by a tool from SHA256 content 220C123F011F8B6414B5F23EA445F84A68223806223CB136A7F0850B87874035
+' Generated by a tool from SHA256 content 56D52FE2A84ABA8B2136FBC6F8DB2D38E8CD88FC95C48E28305B64CC50451321
```

Comment:
>56D52FE2A84ABA8B2136FBC6F8DB2D38E8CD88FC95C48E28305B64CC50451321 [](start = 42, length = 64)

Can this change be reverted?

Reply:
I had to regenerate the files because of new warning error code.
CI checks that those signatures match across files, I think.

---
In reply to: [173945699](https://github.com/dotnet/roslyn/pull/25396#discussion_r173945699) [](ancestors = 173945699)

Reply:
> CI checks that those signatures match across files, I think.

You didn't make any changes for VB. I don't see any good reason for CI system to block the change because these files.

---
In reply to: [173947447](https://github.com/dotnet/roslyn/pull/25396#discussion_r173947447) [](ancestors = 173947447,173945699)